fastlane/Fastfile

```diff
@@ -103,20 +113,22 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
   # Example:
   # fastlane update_appstore_strings version:10.3
   #####################################################################################
-  desc "Updates the PlayStoreStrings.po file"
+  desc 'Updates the PlayStoreStrings.po file'
   lane :update_appstore_strings do |options|
-    prj_folder = Dir.pwd + "/.."
+    prj_folder = "#{Dir.pwd}/.."
 
     files = {
-      release_note: prj_folder + "/Simplenote/metadata/release_notes.txt",
-      play_store_promo: prj_folder + "/Simplenote/metadata/short_description.txt",
-      play_store_desc: prj_folder + "/Simplenote/metadata/full_description.txt",
-      play_store_app_title: prj_folder + "/Simplenote/metadata/title.txt"
+      release_note: "#{prj_folder}/Simplenote/metadata/release_notes.txt",
+      play_store_promo: "#{prj_folder}/Simplenote/metadata/short_description.txt",
+      play_store_desc: "#{prj_folder}/Simplenote/metadata/full_description.txt",
+      play_store_app_title: "#{prj_folder}/Simplenote/metadata/title.txt"
```

Comment:
💄  Use `File.join` instead

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/3267c3d07bb4efb667be614d849bb73e9b2b9ac3